tests/lib/external/snapd-testing-tools/tools/tests.systemd

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: tests.systemd create-and-start-unit <UNIT-NAME> <UNIT-COMMAND-LINE> <UNIT-OVERRIDE>"
+    echo "       tests.systemd stop-and-remove-unit <UNIT-NAME>"
+    echo "       tests.systemd wait-for-service <UNIT-NAME> <UNIT-STATE>"
+}
+
+# Create and start a persistent systemd unit that survives reboots. Use as:
+#   systemd_create_and_start_unit "name" "my-service --args"
+# The third arg supports "overrides" which allow to customize the service
+# as needed, e.g.:
+#   systemd_create_and_start_unit "name" "start" "[Unit]\nAfter=foo"
+create_and_start_unit() {
+    local name=$1
+    local start_line=$2
+    local override=$3
+
+    if [ -z "$name" ]; then
+        echo "tests.systemd: unit name cannot be empty"
+        return 1
+    fi
+    if [ -z "$start_line" ]; then
+        echo "tests.systemd: unit command line cannot be empty"
+        return 1
+    fi
+    if [ -f "/etc/systemd/system/$name.service" ]; then
+        echo "tests.systemd: unit service file already exist"
+        return 1  
+    fi
+
+    printf '[Unit]\nDescription=Support for test %s\n[Service]\nType=simple\nExecStart=%s\n[Install]\nWantedBy=multi-user.target\n' "unknown" "$start_line" > "/etc/systemd/system/$name.service"
```

Comment:
Instead of hardcoding `"unknown"`, maybe we can use `$name` here?

Reply:
I think so, thanks for the suggestion